packages/template-retail-react-app/app/components/social-login/index.jsx

```diff
@@ -64,17 +74,22 @@ const SocialLogin = ({idps}) => {
                         const config = IDP_CONFIG[name.toLowerCase()]
                         const Icon = config?.icon
                         const message = formatMessage(config?.message)
-
                         return (
                             config && (
                                 <Button
-                                    onClick={() => {
-                                        alert(message)
+                                    onClick={async () => {
+                                        // Save the path where the user logged in
+                                        setSessionJSONItem('returnToPage', window.location.pathname)
+                                        // Set local storage item to indicate this is a external login
+                                        window.localStorage.setItem('isSocialProfile', true)
```

Comment:
Hey @bendvc - Wanted to get your input on this. Nate, Pratik, and I have been discussing ways to resolve the "Social Login first" issue, where profiles that have been **created** through Social Login are unable to edit any profile credentials, because well, there aren't any. 

For a short-term solution, I was thinking we could use local storage to denote that the current user is a social login user, and if so, disable edit in the `Account` page. I think in future work we're hoping the `getCustomer` endpoint will somehow be able to indicate whether the user profile was created through an external IDP or ECOM.

Here's how the account page would look like for social login users:
<img width="1417" alt="Screenshot 2024-11-18 at 4 24 23 PM" src="https://github.com/user-attachments/assets/dbb6b928-442f-4055-ac5f-4c7a72dbb58a">